package-kubeapps-version.sh

```diff
@@ -96,7 +96,9 @@ create_release() {
   git commit -m "Adding $tag files"
   info "Committing files and tagging $tag for release"
   git tag "$tag" -m "$tag"
-  git push upstream "tags/$tag"
+  # Pick first remote to push
+  local git_origin="$(git remote | head -n 1)"
```

Comment:
It depends on each running machine. As origin name, some have `origin`, others `upstream`, etc.